e2e/manifests/testnet.toml

```diff
@@ -1,8 +1,48 @@
-network       = "testnet"
+network = "testnet"
 
 public_chains = ["holesky"]
-avs_target    = "holesky"
+avs_target = "holesky"
 
-only_monitor = true
-prometheus   = true
+multi_omni_evms = true
+prometheus = true
 explorer = true
+
+[node.validator01]
+[node.validator02]
+[node.validator03]
+[node.validator04]
+
+[node.seed01]
+mode = "seed"
+
+[node.fullnode01]
+mode = "archive"
+
+
+[keys.validator01]
+validator = "0x5e7B5b243824E15796c33a8829F7807F2C5EeDc0"
+p2p_consensus = "C98F9DFD1F4C94D1E78F74F51BB9E2276B60F0EB"
+
+[keys.validator01_evm]
+p2p_execution = "0x9552600f1331E2d7902C7Fe1b3080136a66594ce"
+
+[keys.validator02]
+validator = "0xD10A6A65C5586aF6093A4F993C3d609758c2a32b"
+p2p_consensus = "29901ADBFD0E8508408B667ACD1AF06DD7C73E88"
+
+[keys.validator02_evm]
```

Comment:
lets keep long lived keys to a minimum:
- validators: only validator keys
- seed nodes: p2p_* keys

I do not think other keys need to be long lived.